development_utilities/gen_unit_test_template.py

```diff
@@ -111,6 +113,17 @@ def generate_json_test_data_for_field(field_type: Type):
         value = bool(random.randint(0, 1))
     elif field_type is float:
         value = random.random() * 9000.0
+    elif field_type is AwareDatetime:
+        # Generate a timezone-aware datetime
+        value = datetime.datetime(
+            year=random.randint(2020, 2023),
+            month=random.randint(1, 12),
+            day=random.randint(1, 28),
+            hour=random.randint(0, 23),
+            minute=random.randint(0, 59),
+            second=random.randint(0, 59),
+            tzinfo=datetime.timezone(datetime.timedelta(hours=random.randint(-12, 14))),
+        )
     elif field_type is datetime.datetime:
```

Comment:
Shorter: 

`elif field_type in (AwareDatetime, datetime.datetime):`

Reply:
And I guess you'd also like to keep the larger range of possible timezone offsets.